src/interfaces/IIrm.sol

```diff
@@ -7,6 +7,7 @@ import {MarketParams, Market} from "./IMorpho.sol";
 /// @author Morpho Labs
 /// @custom:contact security@morpho.org
 /// @notice Interface that Interest Rate Models (IRMs) used by Morpho must implement.
+/// @dev It is the responsibility of the IRM to handle the compounding of interest if desired.
```

Comment:
The compounding of interest is already handled by Morpho Blue, see the use of the `wTaylorCompounded` function. It is the compounding of the variation of the rate that is not necessarily taken into account. The issue was that the `speed` of the variation was assumed to be constant in the IRM, so its effect on the rate would not compound

Reply:
In this case I would remove entirely the comment since it depends even more on how the IRM is built, wdyt?

Reply:
Actually yes, now that I see it I think we should remove it